src/applications/mhv/medications/sass/prescription-details.scss

```diff
@@ -14,4 +14,44 @@
 
 .non-va-ul {
   list-style: none;
+}
+
+.refillProcessIcon {
+  padding: 4px 20px 4px 28px;
+  position: relative;
+  display: inline-block;
+}
+.refillProcessIcon:before {
+  font-family: "Font Awesome 5 Free";
+  font-weight: 700;
+  font-size: 120%;
+  content: "\f56e";
+  position: absolute;
+  left: 0px;
+}
+.unknownIcon {
+  padding: 4px 20px 4px 28px;
+  position: relative;
+  display: inline-block;
+}
+.unknownIcon:before {
+  font-family: "Font Awesome 5 Free";
+  font-weight: 700;
+  font-size: 120%;
+  content: "\f071";
+  position: absolute;
+  left: 0px;
+}
+.submittedIcon {
+  padding: 4px 20px 4px 28px;
+  position: relative;
+  display: inline-block;
+}
+.submittedIcon:before {
+  font-family: "Font Awesome 5 Free";
+  font-weight: 700;
+  font-size: 120%;
+  content: "\f572";
+  position: absolute;
+  left: 0px;
```

Comment:
Styles are duplicated 3 times, could you please refactor? Thanks!

Reply:
updated